services/director/src/simcore_service_director/core/application.py

```diff
@@ -73,6 +74,7 @@ def create_app(settings: ApplicationSettings) -> FastAPI:
         initialize_fastapi_app_tracing(app)
 
     # ERROR HANDLERS
+    set_app_default_http_error_handlers(app)
```

Comment:
All of them will be wrapped? Might break legacy dynamic backend, but I'd have to check.

Reply:
also I believe the status code is still the smae